### ISSUE TITLE
Name attribute unexpectedly being used as accName for elements

### ISSUE THREAD

**Issue by scottaohara** [FreedomScientific/standards-support#725]
In testing the chrome canary implementation of using the `name` attribute to associate `details` elements [HTML PR 9400 having more context for this update](https://github.com/whatwg/html/pull/9400), JAWS 2022 with chromium browsers (not firefox) were using the `name` attribute to give the `details` an accessible name.

This resulted in the following reduced code example being announces as multiple "test" groups.

```
<details name=test>...</details>
<details name=test>...</details>
<details name=test>...</details>
```

This does not appear to be reflected in the a11y tree of chrome, and latest NVDA, for instance, did not announce these elements as being named, either.

Further testing shows that it isn't just `details` that this was happening with, but other HTML elements which can be given an accName (but not all) would inconsistently get an accName from the `name` attribute.  Leading me to believe there is something specific with JAWS/Chrome and that maybe this is an attempt to mitigate author error (maybe for those thinking `name` will "name" an element)?

### Expected result

Do not expose the value of a `name` attribute as the accName for an element.  Particularly `details` elements as with the HTML change, it will become expected to use `name` on these elements for a purpose that is _not_ to provide the element an accName.

### Example

[A test case that demonstrates the issue.](https://codepen.io/scottohara/pen/gOQmeVx)

#### JAWS version and build number
JAWS 2022 (don't have build number on me at this time) - i realize i'm not using the latest JAWS version - so I realize there's a possibility that the latest version doesn't replicate, which would be great.  but, this seems rather specific/potentially on purpose, so not too sure...

 
#### Operating System and version

Win 11

#### Browser and version:
Chrome canary latest

**Comment by stevefaulkner** [comment]
Thanks @scottaohara will test in latest

**Comment by stevefaulkner** [comment]
@scottaohara  tested with JAWS 2022 could reproduce. Then tested with  JAWS 2303.2306.38 (June update) and chrome Version 114.0.5735.199 (Official Build) (64-bit). could not reproduce.

**Comment by scottaohara** [comment]
thank you @stevefaulkner
